mitmproxy/tools/web/master.py

```diff
@@ -125,6 +125,12 @@ def run(self):  # pragma: no cover
                     "No web browser found. Please open a browser and point it to {}".format(web_url),
                     "info"
                 )
+        unknown_opts = optmanager.load_paths(app.CONFIG_PATH)
```

Comment:
This should already happen by default [here](https://github.com/mitmproxy/mitmproxy/blob/8526ca9e175af4a903348c39928535ae10f7543d/mitmproxy/tools/main.py#L69) if I understand things correctly, is that not the case?